packages/test/test-version-utils/src/compatConfig.ts

```diff
@@ -130,171 +126,59 @@ const genLTSConfig = (compatVersion: number | string): CompatConfig[] => {
     ];
 };
 
-// set it in the env for parallel workers
```

Comment:
this all just got put into the lazy evaluated var configList. This ensures it is executed at least and only once. not sure why the diff is so terrible